pages/common/hg-serve.md

```diff
@@ -0,0 +1,27 @@
+# hg serve
+
+> Start a standalone Mercurial web server for browsing repositories.
+
+- Start a web server instance:
+
+`hg serve`
+
+- Start a web server instance on the specified port:
+
+`hg serve --port {{port}}`
+
+- Start a web server instance on the specified listening address:
+
+`hg serve --address {{address}}`
+
+- Start a web server instance with a specific name:
```

Comment:
What does it mean for a web server to have a name ?

Reply:
The name is what appears in the title of the pages. And also it is shown in tabs.

I wasn't quite sure of a good way to shorten it.

Reply:
How about "identifier" or "label" ?